src/main/java/com/elytradev/concrete/block/ConstructionBehaviour.java

```diff
@@ -0,0 +1,79 @@
+package com.elytradev.concrete.block;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.Item;
+import net.minecraft.util.BlockRenderLayer;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import java.util.function.Supplier;
+
+/**
+ * An interface used to describe the behaviour for constructing a {@link ConcreteBlock}.
```

Comment:
Again, this javadoc doesn't help explain anything. It tells us what we already know.

Also, this class name doesn't really seem to fit what this does, which is provide premade subclasses for certain specializations.